src/Murder.Editor/Utilities/OperatingSystemHelpers.cs

```diff
@@ -45,11 +51,16 @@ private static unsafe void Set(void* userdata, byte* text)
             int len = 0;
             while (text[len] != 0) len++;
 
-            string? result = Encoding.UTF8.GetString(text, len);
-
+            string? result = Encoding.UTF8.GetString(text, len) ?? string.Empty;
+            
             if (OperatingSystem.IsMacOS())
             {
-                SetTextForOsx(result ?? string.Empty);
+                SetTextForOsx(result);
+            }
+            
+            if (OperatingSystem.IsLinux())
+            {
+                SDL_SetClipboardText(result);
```

Comment:
Is there a way to not make this crash? I try to make crash as hard as possible to happen in the editor (even on unexpected stuff). I think the way I workarounded this for Mac was wrapping a method around a `try {} catch {}`, so maybe this will work? (if there's a GameLogger.Error message is even better!)